benchmarks/bench_fringefit.py

```diff
@@ -0,0 +1,213 @@
+import os
+import sys
+import shutil
+import unittest
+import itertools
+import numpy as np
+
+# For information about parameters that are unexpectedly zero, set
+# VERBOSE to true.  Currently there are none, so this is for developers
+# only
+VERBOSE = False
+from casatools import ms, ctsys, table
+from casatasks import fringefit, flagmanager, flagdata
+
+from casatestutils import testhelper as th
+ctsys_resolve = ctsys.resolve
+
+tblocal = table()
+
+# ASV iteration control (https://asv.readthedocs.io/en/stable/benchmarks.html#benchmark-attributes)
+number = 1            # i.e., always run the setup and teardown methods
+repeat = (1, 2, 30.0) # between 1 and 2 iterations per round w/ soft cutoff (start no new repeats) past 1m
+rounds = 3            # amount of instances a "repeat block" is run to collect samples
+min_run_count = 5     # enforce the min_repeat * rounds setting is met
+timeout = 3600        # conservative 1hr hard cap for duration of a single test execution
+
+datapath = ctsys.resolve('unittest/fringefit/')
```

Comment:
Please rename the class to a name without the task name as the file already have it. This is to make the appearance of names in the ASV menu page shorter and more descriptive. For example call it:
class SingleBandDelay() using Camel case style. Add any test case from the original script that uses sbd cal tables to this class. Do the same for the next classes and try to group together common test cases.